publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java

```diff
@@ -245,6 +246,9 @@ private Publication fetchExistingPublication(RequestInfo requestInfo, SortableId
 
             throw new ForbiddenException();
         }
+        if (!isThesis(publication) && userCanEditAllNonDegreePublications(requestInfo)) {
```

Comment:
This conditional is not needed, we throw same exception in the end of the method.

Reply:
1. You can also remove unnecessary log messages in that method.
2. You can remove unnecessary condtitional:
```
        if (isThesis(publication) && userUnauthorizedToPublishThesis(requestInfo) && !isOwner) {
            logger.info("thesis={} unauthorized={} isOwner={}",
                        isThesis(publication),
                        userUnauthorizedToPublishThesis(requestInfo),
                        isOwner);

            throw new ForbiddenException();
        }
```
3. This method is over 30 code lines, maybe possible to make it shorter?

Reply:
Refactored, so that the order of the if-statements in the code is irrelevant.